src/axom/spin/MortonIndex.hpp

```diff
@@ -147,7 +151,6 @@ struct MortonBase
    * so, e.g. in 2D, 6 == 0b0110 becomes 0b*0*1*1*0 == 0b00010100 == 20
    */
   AXOM_HOST_DEVICE
-  AXOM_SUPPRESS_UBSAN
```

Comment:
With the new static_assert (a few lines up) and the changes to the BVH generation, I was able to remove this attribute.